### PR TITLE
[BrokenLinksH2] fixing broken links

### PR DESCRIPTION
**Pull request by StephenBrentPeters**

### CODE REVIEW

**Comment by StephenBrentPeters** [comment]
Found quite a few absolute links to turn to relative links.  Working on this.

**Comment by StephenBrentPeters** [comment]
@Rick-Anderson

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. Please review within five business days and merge, or comment in the PR with any changes you'd like to see. 

Several of these files have links that go to sample code that does not seem to exist any longer.  I did my best to find something useful to replace them, and if I couldn't I removed them.  I also replace a large number of absolute links with relative links.

Thanks!

File: aspnet/SameSite/owin-samesite.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ The 2019 draft of the `SameSite` specification:
 ## Supporting older browsers
 
 The 2016 `SameSite` standard mandated that unknown values must be treated as `SameSite=Strict` values. Apps accessed from older browsers which support the 2016 `SameSite` standard may break when they get a `SameSite` property with a value of `None`. Web apps must implement browser detection if they intend to support older browsers. ASP.NET doesn't implement browser detection because User-Agents values are highly volatile and change frequently. An extension point in [ICookieManager](/previous-versions/aspnet/dn800238(v%3Dvs.113)) allows plugging in User-Agent specific logic.
-<!-- https://docs.microsoft.com/previous-versions/aspnet/dn800238(v%3Dvs.113) -->
+<!-- /previous-versions/aspnet/dn800238(v%3Dvs.113) -->
```
Comment (Rick-Anderson) [inline aspnet/SameSite/owin-samesite.md:79]: Revert this change. It's in a comment where the complete URL is more convenient.

**Review by Rick-Anderson — APPROVED** [review]
approve after revert

**Review by StephenBrentPeters — COMMENTED** [review]
performed requested edit
